spec/models/labware_creators/partial_stamped_plate_without_dilution_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'labware_creators/base'
+require_relative '../../support/shared_tagging_examples'
+require_relative 'shared_examples'
+
+# Uses a custom transfer template to transfer material into the new plate
+RSpec.describe LabwareCreators::PartialStampedPlateWithoutDilution do
+  it_behaves_like 'it only allows creation from plates'
+  it_behaves_like 'it has no custom page'
+
+  has_a_working_api
+
+  let(:well_a1) do
```

Comment:
I think it might be a good idea to have a well which has several requests for different library types so there is some overlap like a real chromium example. 
Plus maybe a well with NO requests.
To give confidence the code still selects the right wells for a given library type filter.

These wells defined here all have one request each.